### ISSUE TITLE
cancelButtonStyle used both in <Animated.View> an <Text> causes warnings for some style properties

### ISSUE THREAD

**Issue by j-nolan** [crabstudio/react-native-search-box#18]
Hello,

It seems like the `cancelButtonStyle` prop is passed in two components:
- `<Animated.View>` at [line 365](https://github.com/crabstudio/react-native-search-box/blob/master/index.js#L365)
- `<Text>` at [line 372](https://github.com/crabstudio/react-native-search-box/blob/master/index.js#L372)

This causes warnings when we uses style properties that are not compatible with bot `<Animated.View />` and <Text />`.

Example:
```jsx

<SearchBar
  cancelButtonStyle={{ fontSize: 20 }}
/>
```

Will show a warning:
> Warning: Failed prop type: Invalid props.style key `fontSize` supplied to `RCTView`.

My suggestion would be to make two new props (`cancelButtonViewStyle` and `cancelTextButtonStyle`) to customize <Animated.View /> and `<Text>`. The `cancelButtonStyle`  should throw a deprecation warning. This would be backward compatible.

I'd be happy to make a PR if this makes sense

**Comment by anhtuank7c** [comment]
@j-nolan Yes i am, use `cancleButtonStyle` to both `View, Text` [L348](https://github.com/agiletechvn/react-native-search-box/blob/master/index.js#L348)
So impossible to using `fontX` props in `cancleButtonStyle`
